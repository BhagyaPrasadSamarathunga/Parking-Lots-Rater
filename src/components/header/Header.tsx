
const Header = (): JSX.Element => {

  return (
    <div className="flex justify-between items-center h-[52px] lg:h-[60px] z-20 fixed w-full top-0 left-0 bg-sky-600">
      <p className="text-white font-semibold ml-6 text-lg">Park Rater</p>
    </div>
  )
}

export default Header;