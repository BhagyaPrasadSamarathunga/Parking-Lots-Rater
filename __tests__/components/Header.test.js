import React from 'react';
import { render, screen } from '@testing-library/react';
import Header from '../../src/components/header/Header';

describe('Header component', () => {

  test('renders with correct label', () => {
    render(<Header />);
    const headerElement = screen.getByText(/Park Rater/i);
    expect(headerElement).toBeInTheDocument();
  });

  test('matches snapshot', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});