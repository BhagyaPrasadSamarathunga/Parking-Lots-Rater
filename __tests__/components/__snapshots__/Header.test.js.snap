// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex justify-between items-center h-[52px] lg:h-[60px] z-20 fixed w-full top-0 left-0 bg-sky-600"
  >
    <p
      class="text-white font-semibold ml-6 text-lg"
    >
      Park Rater
    </p>
  </div>
</DocumentFragment>
`;
